ultralytics/nn/modules/head.py

```diff
@@ -296,7 +298,10 @@ def forward(self, x):
         if isinstance(x, list):
             x = torch.cat(x, 1)
         x = self.linear(self.drop(self.pool(self.conv(x)).flatten(1)))
-        return x
+        if self.training:
```

Comment:
@Laughing-q This is how it was before. The problem here is that `self.training` is False during validation, so the validation losses are not correct.

Reply:
Ah okay, nice.